website/docs/r/data_factory_trigger_tumbling_window.html.markdown

```diff
@@ -0,0 +1,133 @@
+---
+subcategory: "Data Factory"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_data_factory_trigger_tumbling_window"
+description: |-
+  Manages a Tumbling Window Trigger.
+---
+
+# azurerm_data_factory_trigger_tumbling_window
+
+Manages a Tumbling Window Trigger inside an Azure Data Factory. 
+
+Datafactory triggers are created in the "Stopped" state and must be manually enabled to start triggering.  The API prevents updates to triggers in the "Started" state, so they must be manually paused before running terraform apply. 
+
+## Example Usage
+
+```hcl
+
+resource "azurerm_data_factory_trigger_tumbling_window" "example" {
+  name                = "example"
+  data_factory_name   = "example"
+  resource_group_name = "example"
+
+  pipeline_parameters = {
+    example = "@{formatDateTime(trigger().outputs.windowStartTime,'yyyy-MM-dd')}"
```

Comment:
could we pass this in formatted instead of this rather unclear series of functions